### PR TITLE
Update PyTorch pin

### PR DESCRIPTION
**Pull request by anmyachev**
CI: 
* https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/11041821091/job/30672924709
* https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/11053414365 (more recent pytorch commit 963e793e1b9e7adbce66162c6e3b117343386d37) **Failed** Another error: 
`FAILED [0.0046s] inductor/test_triton_heuristics.py::TestTritonHeuristics::test_autotune_hints_to_configs - AssertionError: Torch not compiled with CUDA enabled`
* https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/11054121832 (190e09d8b6a13f789b143f0fbd1325f924550967, in progress)
https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/11055121246

### CODE REVIEW

**Comment by alexbaden** [comment]
hit by this issue again:
```
FAILED [3.8733s] inductor/test_triton_kernels.py::KernelTests::test_triton_kernel_dtype_view_cfg_cpp_abi - torch._dynamo.exc.BackendCompilerFailed: backend='inductor' raised:
AttributeError: 'NoneType' object has no attribute 'push_codegened_graph'
```

File: .github/pins/pytorch-upstream.txt [hunk 1]
```diff
@@ -1 +1 @@
-9fd54d787d9ee426c9165376ee6add0ef731b07b
+190e09d8b6a13f789b143f0fbd1325f924550967
```
Comment (anmyachev) [inline .github/pins/pytorch-upstream.txt:1]: @alexbaden @vlad-penkin we can update pytorch to this commit with one patch: https://github.com/pytorch/pytorch/pull/136761. The tests are passing, is this suitable for us?

Comment (alexbaden) [reply .github/pins/pytorch-upstream.txt:1]: sounds good to me

**Review by alexbaden — APPROVED** [review]

File: .github/pins/pytorch-upstream.txt (hunk 1 above)
Comment (vlad-penkin) [reply .github/pins/pytorch-upstream.txt:1]: agree

**Review by vlad-penkin — APPROVED** [review]

**Pull request merged**